scripts/lbnl_group.nhc

```diff
@@ -0,0 +1,40 @@
+# LBNL NHC - Group/members check
+#
+# Andy Georges <andy.georges@ugent.be>
+# 20 April 2018
+
+GETENT_CMD="${GETENT:-getent}"
+
+# syntax: check_group_member [-t timeout] -g group
+function check_group_member() {
+    local TIMEOUT=10
+    local GROUP="" WHOLEGROUP=""
+
+    OPTIND=1
+    while getopts ":t:g:" OPTION; do
+        case "$OPTION" in
+            t) TIMEOUT="$OPTARG" ;;
+            g) GROUP="$OPTARG" ;;
+            -) ((OPTIND++)) ; break ;;
+            :) die 1 "$CHECK: Option -$OPTARG requires an argument."; return 1 ;;
+            \?) die 1 "$CHECK: Invalid option: -$OPTARG"; return 1 ;;
+        esac
+    done
+
+    WHOLEGROUP=$($GETENT_CMD group "$GROUP" 2>/dev/null)
```

Comment:
we should really check if this works or not on skitty/.... (see the enumerate stuff). otherwise the correct python code is in vsc-base
@JensTimmerman what was it again that worked?

Reply:
@stdweird the python code is in https://github.com/hpcugent/vsc-base/pull/255
but this just works in bash

listing all groups a user belongs to works with:
`groups vsc40485`
listing all users in a specific group works with
`getent group gvo00002`